openflow_input/bsn_tlv

```diff
@@ -1077,8 +1077,25 @@ struct of_bsn_tlv_force_link_up : of_bsn_tlv {
     uint16_t length;
 };
 
+struct of_bsn_tlv_rest_server : of_bsn_tlv {
+    uint16_t type == 152;
+    uint16_t length;
+};
+
+struct of_bsn_tlv_uri_scheme : of_bsn_tlv {
+    uint16_t type == 153;
+    uint16_t length;
+    of_octets_t value;
+};
+
 struct of_bsn_tlv_timestamp : of_bsn_tlv {
     uint16_t type == 154;
     uint16_t length;
     uint64_t value;
 };
+
+struct of_bsn_tlv_record_packets : of_bsn_tlv {
+    uint16_t type == 155;
+    uint16_t length;
+    uint8_t value;
```

Comment:
uint8_t should be plenty, but I can change to a uint32_t just to be safe if you'd like